lib/remote_console/server_adapter/websocket_binary.rb

```diff
@@ -0,0 +1,42 @@
+module RemoteConsole
+  module ServerAdapter
+    class WebsocketBinary
+      attr_reader :env, :url
+
+      def initialize(env, sock)
+        @env = env
+        @sock = sock
+
+        secure = Rack::Request.new(env).ssl?
+        scheme = secure ? 'wss:' : 'ws:'
+        @url = scheme + '//' + env['HTTP_HOST'] + env['REQUEST_URI']
```

Comment:
- Where are these env vars coming from?
- Might be preferable to use the URI::Generic.build method, as it will handle things like ipv6 and any off escaping and such. cc @jrafanie

Reply:
The env is coming from the Rack request, we had this discussion with @jrafanie and we ended up with not changing it. You can see the discussion [here](https://github.com/ManageIQ/manageiq/pull/18093/files#r239643392) and [here](https://github.com/ManageIQ/manageiq/pull/18287). Again, this is just moving code, so I don't think it's okay to change the behavior in any way.

Reply:
Yeah, because of the way GitHub shows the transfer, I didn't realize it was a rename + remodulizing.  As such, ignore my comments for this PR, and think about them for a follow up.

Reply:
@Fryguy okay, can we get this in then? :wink: